libs/shared/devops-copilot/feature/src/lib/hooks/use-config/use-config.tsx

```diff
@@ -0,0 +1,37 @@
+import { useQuery } from '@tanstack/react-query'
+import { queries } from '@qovery/state/util-queries'
+
+export interface Config {
+  id: string
+  title: string
+  created_at: string
+  updated_at: string
+  organization_id: string
+}
+
+interface UseConfigReturn {
+  enabled: boolean
+  readOnly: boolean
+  instructions: string
+  isLoading: boolean
+  error: string | null
+  refetchConfig: () => Promise<void>
+}
+
+export const useConfig = ({ organizationId }: { organizationId: string }): UseConfigReturn => {
```

Comment:
The hook should follow the same format as your other return hooks, it should expose all the parameters returned by `useQuery`. So we can reuse all values provided by React Query

For example, it should be usable like this:

```ts
const { data: readOnly } = useConfig({
  organizationId: context?.organization?.id ?? '',
})

const isReadOnly = readOnly?.org_config?.read_only ?? true
```

Could you fix it ? 🙏